### PR TITLE
Adding SMS notification template configurations

### PR DESCRIPTION
**Pull request by somindatommy**
## Purpose of the PR

- Adding SMS notification template configurations.
- PR should merge after merging https://github.com/wso2-extensions/identity-event-handler-notification/pull/110

### CODE REVIEW

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 1]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+4. Click on the **sms directory** and click **Add Collection** and enter collection name as 
+`accountconfirmation`. This will create another directory inside the sms directory.
+![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
+5. Click on **Add Resource**.
+6. Select **method** as `Create Text Content`.
+7. Provide **name** as `en_us`.
+8. Add the SMS notification template body. **NOTE:** Include `{{confirmation-code}}` placeholder 
+in the SMS template body.
+![add-template-resource](../assets/img/learn/self-registration/adding-sms-templates/add-template-resource.png)
+9. Click on en_us resource and add the following properties.
```
Comment (sherenewso2) [inline en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Click `en_us` and....

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 2]
```diff
@@ -266,6 +296,9 @@ For more details see,
 
 WSO2 Identity Server provides the functionality to confirm the user account internally or externally.
 
+!!! note
+    To try out self-registration, above steps needs to be completed.
```
Comment (sherenewso2) [inline en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: !!! tip "Before you begin"
    Complete the steps given above before trying out self-registration.

Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 3]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
+3. Click **Add Collection** and enter collection name as `sms`.
+![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+4. Click on the **sms directory** and click **Add Collection** and enter collection name as 
+`accountconfirmation`. This will create another directory inside the sms directory.
+![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
+5. Click on **Add Resource**.
+6. Select **method** as `Create Text Content`.
+7. Provide **name** as `en_us`.
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 4]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
+3. Click **Add Collection** and enter collection name as `sms`.
+![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+4. Click on the **sms directory** and click **Add Collection** and enter collection name as 
+`accountconfirmation`. This will create another directory inside the sms directory.
+![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
+5. Click on **Add Resource**.
+6. Select **method** as `Create Text Content`.
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 5]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
+3. Click **Add Collection** and enter collection name as `sms`.
+![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+4. Click on the **sms directory** and click **Add Collection** and enter collection name as 
+`accountconfirmation`. This will create another directory inside the sms directory.
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 6]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
+3. Click **Add Collection** and enter collection name as `sms`.
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed:d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 7]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+4. Click on the **sms directory** and click **Add Collection** and enter collection name as 
+`accountconfirmation`. This will create another directory inside the sms directory.
+![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 8]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
+3. Click **Add Collection** and enter collection name as `sms`.
+![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 9]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
+2. In **tree view tab**, click `system>config>identity`.
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 10]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+    customize the email template. For more information, 
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
+
+1. Login to the management console and navigate to `Main> Registry> Browse`.
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 11]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
...
+!!! tip
+    The **AccountConfirmation** template is used to send email notifications.You can edit and 
+    customize the email template. For more information, 
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configs are case sensitive. Changing the case might break the functionality
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 12]
```diff
@@ -256,6 +249,43 @@ Management Policies** section.
         </tbody>
         </table>
         
+## Adding SMS notification templates
+
+The templates for the EMAIL notification are stored in `<IS_HOME</repository/conf/email
```
[the remark this answers is not in the document]
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: d72b99b72fc947f18610188db171158cd99f8f08

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 13]
```diff
@@ -256,6 +249,50 @@ Management Policies** section.
...
+    see [Customizing Automated Emails](../../learn/customizing-automated-emails).
+    
+The templates for SMS notifications are stored in the registry. Follow the steps below to add
+SMS notification templates.
+
+!!! Warning
+    All the configurations are case sensitive. Changing the case may break the functionality and 
+    cause errors.
+
+1. Log in to the Management Console and click `Main> Registry> Browse`.
```
Comment (sherenewso2) [inline en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Please make Main > Registry > Browse **bold** and not ` monospace `

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md [hunk 14]
```diff
@@ -256,6 +249,50 @@ Management Policies** section.
...
+2. On the **tree view** tab, click `system -> config ->identity`.
+
+3. Click **Add Collection**, enter `sms` as the **Name**, and then click **Add**.
+    ![add-sms-collection](../assets/img/learn/self-registration/adding-sms-templates/add-sms-collection.png)
+    
+4. Click `sms` and then click **Add collection**. Enter `accountconfirmation` as the **Name. 
+This will create another directory inside the sms directory.
+    ![add-accountconfirmation-collection](../assets/img/learn/self-registration/adding-sms-templates/add-accountconfirmation-collection.png)
+    
+5. Click on **Add Resource**.
```
Comment (sherenewso2) [inline en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: The content after "Name" should not be in Bold. Only "Add Resource" in step 5 should be bold.

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (sherenewso2) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Please make Main > Registry > Browse bold and not `monospace`

File: en/docs/learn/multi-channel-self-registration-and-account-confirmation.md (hunk 14 above)
Comment (somindatommy) [reply en/docs/learn/multi-channel-self-registration-and-account-confirmation.md]: Fixed: 933ce08ecb17d0941395f9188251ca65508db919

**Pull request merged**